lib/kb_Metrics/metricsDBs.py

```diff
@@ -34,12 +34,16 @@ def __init__(self, mongo_host, mongo_dbs, mongo_user, mongo_psswd):
         self.metricsDBs = dict()
         for m_db in mongo_dbs:
             # create the client and authenticate
-            self.mongo_clients[m_db] = MongoClient(
-                "mongodb://" + mongo_user + ":" + mongo_psswd +
-                "@" + mongo_host + "/" + m_db)
+            if mongo_host:
+                self.mongo_clients[m_db] = MongoClient(
+                    "mongodb://" + mongo_user + ":" + mongo_psswd +
+                    "@" + mongo_host + "/" + m_db)
+            else: # default to localhost:27017
```

Comment:
This will never happen in production. I don't think you should do this just for local tests.

Reply:
Did you mean the 'else' part?  Yes, I added it here just for local tests.

Reply:
@qzzhang yea. I don't think you should add a case to prod code base just for local tests.

Reply:
Because I need to test the functions in metricsDBs.py.  Should I remove it?